form_ong.html

```diff
@@ -40,16 +40,16 @@ <h2 class="titulo-cadastro-um">Dados da ONG</h2>
                     <div id="campos-ong">
 
                         <div class="flex-inner">
-                            <input type="text" id="nome-grupo-ong" name="nomeGrupoOng" placeholder="Nome do Grupo/Organização" class="w40"/>
+                            <input type="text" id="nome-grupo-ong" name="nomeGrupoOng" placeholder="Nome do Grupo/Organização"/>
                             <input type="text" id="cnpj-ong" name="cnpj-ong" placeholder="CNPJ" class="w40"/>
```

Comment:
Ficou faltando retirar a classe desse elemento.
```suggestion
                            <input type="text" id="cnpj-ong" name="cnpj-ong" placeholder="CNPJ"/>
```